Dockerfile

```diff
@@ -46,6 +55,16 @@ COPY --link frankenphp/conf.d/10-app.ini $PHP_INI_DIR/app.conf.d/
 COPY --link --chmod=755 frankenphp/docker-entrypoint.sh /usr/local/bin/docker-entrypoint
 COPY --link frankenphp/Caddyfile /etc/frankenphp/Caddyfile
 
+RUN set -eux; \
+    groupadd -g $GID $GROUP; \
+    useradd -u $UID -g $GID --no-create-home $USER; \
+    setcap CAP_NET_BIND_SERVICE=+eip /usr/local/bin/frankenphp; \
+    chown -R $UID:$GID /data/ /config/; \
+    mkdir -p var/cache var/log; \
+    chown -R $UID:$GID var/cache var/log
```

Comment:
The image already contains the user `www-data`. This user has all the necessary privileges to run the web server. Would it be better to use it instead of creating a new one?